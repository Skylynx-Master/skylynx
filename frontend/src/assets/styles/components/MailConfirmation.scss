@import "../Vars.scss";
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

.MailConfirmation {
  width: 660px;
  height: 500px;
  margin: 8% auto 0;
  background-color: $color-white;
  padding: 20px;
  text-align: center;
  border-radius: 20px;
}

.MailConfirmation h2 {
  font-size: 35px;
  padding: 50px 0;
}

.MailConfirmation img {
  width: 170px;
  height: 170px;
}
