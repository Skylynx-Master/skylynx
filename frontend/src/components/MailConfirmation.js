import React from "react";
import "../assets/styles/components/MailConfirmation.scss";
import mailConfirmation from "../assets/static/MailConfirmationIcon.png";

const MailConfirmation = () => {
  return (
    <div className="MailConfirmation">
      <h2 className="MailConfirmation-title">
        Verifique su correo para confirmar su cuenta
      </h2>
      <img
        className="MailConfirmation-image"
        src={mailConfirmation}
        alt="mail confirmation logo"
      ></img>
    </div>
  );
};

export default MailConfirmation;
