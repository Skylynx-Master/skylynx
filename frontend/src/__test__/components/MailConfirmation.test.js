import React from "react";
import MailConfirmation from "../../components/MailConfirmation";
import { mount } from "enzyme";
import { BrowserRouter } from "react-router-dom";

describe("MailConfirmation Component Testing", () => {
  const mail = mount(
    <BrowserRouter>
      <MailConfirmation />
    </BrowserRouter>
  );
  test("Mail Confirmation Render", () => {
    expect(mail.length).toEqual(1);
  });
  test("Mail Confirmation title Render", () => {
    expect(mail.find(".MailConfirmation-title").length).toEqual(1);
  });

  test("Mail Confirmation title text", () => {
    expect(mail.find(".MailConfirmation-title").text()).toEqual(
      "Verifique su correo para confirmar su cuenta"
    );
  });

  test("Mail Confirmation image render", () => {
    expect(mail.find(".MailConfirmation-image").length).toEqual(1);
  });
});
