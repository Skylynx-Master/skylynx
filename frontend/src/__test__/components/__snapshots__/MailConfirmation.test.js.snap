// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailConfirmation Snapshot Test UI MailConfirmation component 1`] = `
<div
  className="MailConfirmation"
>
  <h2
    className="MailConfirmation-title"
  >
    Verifique su correo para confirmar su cuenta
  </h2>
  <img
    alt="mail confirmation logo"
    className="MailConfirmation-image"
    src=""
  />
</div>
`;
